src/components/ClubHome/Daily/DailyControl.js

```diff
@@ -59,7 +60,43 @@ const DailyControl = ({
                     moveCal();
                   }}
                 />
-                <AiOutlineStar />
+                {el.important ? (
+                  <AiFillStar
+                    onClick={() => {
+                      axios(
+                        `http://3.36.72.145:8080/api/club/schedule/1/${el.no}`,
+                        // `http://3265-218-39-136-26.ngrok.io/api/club/schedule/1/${el.no}`,
+                        {
+                          method: "PATCH",
+                          headers: {
+                            "Content-type": "application/json; charset=utf-8",
+                            "x-auth-token":
+                              "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6InRlc3QxIiwibmFtZSI6InRlc3QxIiwiY2x1Yk51bSI6IlsxXSJ9.1u6k5cJuaUlZj14CJJZiI8guHnlZXf1uuU6vZjl9jNk",
+                          },
+                          data: { important: 0 },
+                        }
+                      ).then((res) => console.log(res));
```

Comment:
```suggestion
                      ).then((res) => console.log(res)).catch((err) => alert(err.response.data.msg));
```
이렇게 에러 잡아주는게 좋을것 같아용